include/srdfdom/trim.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Evan Teran
```

Comment:
I don't think this header should be in the public interface of the package.
I propose to either put it into an anonymous namespace in `src/model.cpp` (it's used only there atm)
or just move the file to `src/` to avoid installing it.

Reply:
Agreed, this shouldn't be in the public interface. I'll move the file to the `src/` directory as you and @rhaschke recommend.